### PR TITLE
Clarify that the default value for the max_allowed_packets setting applies to cloud and on-premises projects

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) clarifies that the default value of 16MB for the MySQL `max_allowed_packets` setting applies to both cloud and on-premises projects.

~The linked KB article is still in draft and has not yet been published.~ (removed because it linked back to this topic for instructions)

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/install-gde/prereq/mysql.html

whatsnew
Added clarification that the default value of 16MB for the MySQL [`max_allowed_packets`](https://devdocs.magento.com/guides/v2.4/install-gde/prereq/mysql.html) setting applies to both Magento Commerce Cloud and on-premises projects.

### CODE REVIEW

File: src/guides/v2.3/install-gde/prereq/mysql.md [hunk 1]
```diff
@@ -89,6 +89,9 @@ To install MySQL 5.7 on Ubuntu 16:
 
 1. If you expect to import large numbers of products into Magento, you can increase the value for [`max_allowed_packet`](http://dev.mysql.com/doc/refman/5.6/en/program-variables.html){:target="_blank"} that is larger than the default, 16MB.
 
+   {:.bs-callout-info}}
+   The default applies to {{site.data.var.ece}} *and* on-premises projects. {{site.data.var.ece}} Pro customers must open a support ticket to increase the default. {{site.data.var.ece}} Starter customers can increase the default in the [`/etc/mysql/my.cnf`](https://support.magento.com/hc/en-us/articles/360051598831) file.
```
Comment (meker12) [inline src/guides/v2.3/install-gde/prereq/mysql.md:93]:
```suggestion
   The default value applies to {{site.data.var.ece}} *and* on-premises projects. {{site.data.var.ece}} Pro customers must open a support ticket to increase the `max_allowed_packet` value. {{site.data.var.ece}} Starter customers can increase the  value by updating the configuration in the [`/etc/mysql/my.cnf`](https://support.magento.com/hc/en-us/articles/360051598831) file.
```

File: src/guides/v2.4/install-gde/prereq/mysql.md [hunk 2]
```diff
@@ -38,6 +38,9 @@ Magento 2.4 requires a clean installation of MySQL 8.0. Follow the links below f
 
 If you expect to import large numbers of products into Magento, you can increase the value for [`max_allowed_packet`](http://dev.mysql.com/doc/refman/5.6/en/program-variables.html){:target="_blank"} that is larger than the default, 16MB.
 
+{:.bs-callout-info}}
+The default applies to {{site.data.var.ece}} *and* on-premises projects. {{site.data.var.ece}} Pro customers must open a support ticket to increase the default. {{site.data.var.ece}} Starter customers can increase the default in the [`/etc/mysql/my.cnf`](https://support.magento.com/hc/en-us/articles/360051598831) file.
```
Comment (meker12) [inline src/guides/v2.4/install-gde/prereq/mysql.md:42]:
```suggestion
The default value applies to {{site.data.var.ece}} *and* on-premises projects. {{site.data.var.ece}} Pro customers must open a support ticket to increase the `max_allowed_packet` value. {{site.data.var.ece}} Starter customers can increase the  value by updating the configuration in the [`/etc/mysql/my.cnf`](https://support.magento.com/hc/en-us/articles/360051598831) file.
```

**Review by meker12 — CHANGES_REQUESTED** [review]

**Review by meker12 — APPROVED** [review]

File: src/guides/v2.4/install-gde/prereq/mysql.md [hunk 3]
```diff
@@ -38,6 +38,9 @@ Magento 2.4 requires a clean installation of MySQL 8.0. Follow the links below f
 
 If you expect to import large numbers of products into Magento, you can increase the value for [`max_allowed_packet`](http://dev.mysql.com/doc/refman/5.6/en/program-variables.html){:target="_blank"} that is larger than the default, 16MB.
 
+{:.bs-callout-info}}
+The default value applies to {{site.data.var.ece}} *and* on-premises projects. {{site.data.var.ece}} Pro customers must open a support ticket to increase the `max_allowed_packet` value. {{site.data.var.ece}} Starter customers can increase the  value by updating the configuration in the [`/etc/mysql/my.cnf`](https://support.magento.com/hc/en-us/articles/360051598831) file.
```
Comment (meker12) [inline src/guides/v2.4/install-gde/prereq/mysql.md:42]:
```suggestion
The default value applies to {{site.data.var.ece}} *and* on-premises projects. {{site.data.var.ece}} Pro customers must open a support ticket to increase the `max_allowed_packet` value. {{site.data.var.ece}} Starter customers can increase the value by updating the configuration in the [`/etc/mysql/my.cnf`](https://support.magento.com/hc/en-us/articles/360051598831) file.
```

File: src/guides/v2.3/install-gde/prereq/mysql.md [hunk 4]
```diff
@@ -89,6 +89,9 @@ To install MySQL 5.7 on Ubuntu 16:
 
 1. If you expect to import large numbers of products into Magento, you can increase the value for [`max_allowed_packet`](http://dev.mysql.com/doc/refman/5.6/en/program-variables.html){:target="_blank"} that is larger than the default, 16MB.
 
+   {:.bs-callout-info}}
+   The default value applies to {{site.data.var.ece}} *and* on-premises projects. {{site.data.var.ece}} Pro customers must open a support ticket to increase the `max_allowed_packet` value. {{site.data.var.ece}} Starter customers can increase the  value by updating the configuration in the [`/etc/mysql/my.cnf`](https://support.magento.com/hc/en-us/articles/360051598831) file.
```
Comment (meker12) [inline src/guides/v2.3/install-gde/prereq/mysql.md:93]:
```suggestion
   The default value applies to {{site.data.var.ece}} *and* on-premises projects. {{site.data.var.ece}} Pro customers must open a support ticket to increase the `max_allowed_packet` value. {{site.data.var.ece}} Starter customers can increase the value by updating the configuration in the [`/etc/mysql/my.cnf`](https://support.magento.com/hc/en-us/articles/360051598831) file.
```

**Review by meker12 — APPROVED** [review]

**Comment by hguthrie** [comment]
So, I wondered if the Cloud guide needed to discuss the `my.cnf` file more, and it seem that it is only mentioned in the older local development set up and in the Docker service containers section. I think we should investigate whether this is worth mentioning where custom configuration of the `my.cnf` file is explained. For example, can this be added to the db:environment section of the Docker override YML? Just curious.

**Comment by meker12** [comment]
> So, I wondered if the Cloud guide needed to discuss the `my.cnf` file more, and it seem that it is only mentioned in the older local development set up and in the Docker service containers section. I think we should investigate whether this is worth mentioning where custom configuration of the `my.cnf` file is explained. For example, can this be added to the db:environment section of the Docker override YML? Just curious.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**